scandium-core/src/test/resources/Scandium-logging.properties

```diff
@@ -69,3 +69,8 @@ org.eclipse.californium.scandium.useParentHandlers = false
 org.eclipse.californium.scandium.level = INFO
 org.eclipse.californium.scandium.dtls.level = CONFIG
 org.eclipse.californium.scandium.dtls.Connection.level = INFO
+
+org.eclipse.californium.elements.handlers = java.util.logging.ConsoleHandler
+org.eclipse.californium.elements.useParentHandlers = false
+org.eclipse.californium.elements.level = INFO
+org.eclipse.californium.elements.util.DirectDatagramSocketImpl.level = FINE
```

Comment:
is it intentional to set this log level to FINE, this make a lot of log on `mvn clean install`.

Reply:
The intention is to see the exchange messages in the log. Yes this make large logs, but without it's not possible to find the cause of failures of rarely failing tests.